specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```diff
@@ -4486,6 +4510,11 @@
           "format": "int32",
           "description": "Routing weight for vpn connection."
         },
+        "dpdTimeoutSeconds": {
```

Comment:
This should be added in a new API version.

Reply:
@majastrz Done

Reply:
Thanks!

---
In reply to: [439132408](https://github.com/Azure/azure-rest-api-specs/pull/9722#discussion_r439132408) [](ancestors = 439132408)